lib/kite-process-error.js

```diff
@@ -4,7 +4,13 @@ module.exports = class KiteProcessError extends Error {
   constructor(type, data) {
     super(type);
     this.data = data;
-    this.message = `${type}: ${data.message}\ncmd: ${data.cmd}\nstdout: ${data.stdout}\nstderr: ${data.stderr}`;
+    this.message = `${type}: ${data.message}\ncmd: ${data.cmd}`;
```

Comment:
While verbose, it may be worth checking that data, data.message, and data.cmd are non-null, if there are other consumers of `KiteProcessError`.

Reply:
merging without this change for now, this was already in before. 
But we definitely should improve this in another PR (I'll create an issue)